docs/add-ons/velero.md

```diff
@@ -0,0 +1,144 @@
+# [Velero](https://velero.io/)
+
+Velero is an open source tool to safely backup and restore, perform disaster recovery, and migrate Kubernetes cluster resources and persistent volumes.
+
+- [Helm chart](https://github.com/vmware-tanzu/helm-charts/tree/main/charts/velero)
+- [Plugin for AWS](https://github.com/vmware-tanzu/velero-plugin-for-aws)
+
+## Usage
+
+[Velero](https://github.com/aws-ia/terraform-aws-eks-blueprints/tree/main/modules/kubernetes-addons/velero) can be deployed by enabling the add-on via the following.
+
+```hcl
+enable_velero           = true
+velero_backup_s3_bucket = "<YOUR_BUCKET_NAME>"
+```
+
```

Comment:
Show the example of how to use custom `values.yaml` for this add-on

Reply:
updated in https://github.com/aws-ia/terraform-aws-eks-blueprints/pull/407/commits/6354998ecf15c55d1f7087f4d5f5d42e6aea06ac